test/fail/ok/type-alias-path.tc.ok

```diff
@@ -0,0 +1,20 @@
+type-alias-path.mo:18.41-18.42: type error [M0096], expression of type
+  {m1 : M1.Map<Nat, Text>; m2 : MyMap.Map<Nat, Text>}
+cannot produce expected type
+  {m2 : M1.Map<Nat, Text>}
+type-alias-path.mo:21.5-21.53: type error [M0098], cannot apply function of type
+  <T>(t : T, T -> ()) -> ()
+to argument of type
+  (M1.Map<Nat, Text>, (m : MyMap.Map<Nat, Text>) -> ())
+because there is no way to satisfy subtyping
+  (m : MyMap.Map<Nat, Text>) -> ()  <:  M1.Map<Nat, Text> -> ()
+type-alias-path.mo:24.24-24.33: type error [M0096], expression of type
+  <K, V>(self : MyMap.Map<K, V>,
+         compare : (implicit : (K, K) -> Types.Order), key : K) ->
+    ?V
+cannot produce expected type
+  () -> ()
+type-alias-path.mo:29.10-29.12: type error [M0096], expression of type
+  {x : M.MyType}
+cannot produce expected type
+  {x : M.MyType}
```

Comment:
It is still possible to create such errors, we cannot avoid that since we can rename modules